### PR TITLE
Feature/multiple-apikey-support

### PR DESCRIPTION
**Pull request by pmpnarola**

### CODE REVIEW

File: xendit-java-lib/src/main/java/com/xendit/XenditClient.java [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+  private XenditClient() {}
+
+  public static class Builder {
+
+    private String apikey;
+
+    public Builder() {}
+
+    public Builder apikey(String apikey) {
```
Comment (stanleynguyen) [inline xendit-java-lib/src/main/java/com/xendit/XenditClient.java:44]: Name this `setApikey` instead?

**Review by stanleynguyen — CHANGES_REQUESTED** [review]
Instead of naming our packages using camelCase (e.g. `cardlessCredit`), can we follow the java convention of lowercase (e.g. `cardlesscredit`)? Btw I wonder if the breaking package path change is really necessary?

Also, we seem to be missing documentation updates?

File: xendit-java-lib/src/main/java/com/xendit/model/cardlessCredit/CardlessCredit.java [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+import java.util.Map;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+@Getter
+@Setter
+@ToString
+@NoArgsConstructor
```
Comment (javiersuweijie) [inline xendit-java-lib/src/main/java/com/xendit/model/cardlessCredit/CardlessCredit.java:16]: why do we need a constructor for this? all the methods are static and exposing a constructor might be confusing.

**Review by javiersuweijie — CHANGES_REQUESTED** [review]
Some nitpicks but generally okay. Agree with stanley that the package should be lower case.

File: xendit-java-lib/src/main/java/com/xendit/XenditClient.java (hunk 1 above)
Comment (pmpnarola) [reply xendit-java-lib/src/main/java/com/xendit/XenditClient.java:44]: Yes @stanleynguyen, I will change variable naming.

File: xendit-java-lib/src/main/java/com/xendit/model/cardlessCredit/CardlessCredit.java (hunk 2 above)
Comment (pmpnarola) [reply xendit-java-lib/src/main/java/com/xendit/model/cardlessCredit/CardlessCredit.java:16]: Yes, package name should be lower case. Will change that. and also will remove constructor if required.

File: xendit-java-lib/src/main/java/com/xendit/XenditClient.java (hunk 1 above)
Comment (pmpnarola) [reply xendit-java-lib/src/main/java/com/xendit/XenditClient.java:44]: > " Btw I wonder if the breaking package path change is really necessary? "

We have created seperate folder for each operation and its client in model package. Instead we can place all operation and its client directly in model package that would not break package path.

Comment (pmpnarola) [reply xendit-java-lib/src/main/java/com/xendit/XenditClient.java:44]: > Name this `setApikey` instead?

Yes. That can also be change.

Comment (pmpnarola) [reply xendit-java-lib/src/main/java/com/xendit/XenditClient.java:44]: > Also, we seem to be missing documentation updates?

Yes. That is pending. We will update it.

Comment (stanleynguyen) [reply xendit-java-lib/src/main/java/com/xendit/XenditClient.java]: > We have created seperate folder for each operation and its client in model package. Instead we can place all operation and its client directly in model package that would not break package path.

Let's do this instead so we don't have to bump major version?

**Review by javiersuweijie — APPROVED** [review]

**Pull request merged**